tekton/resources/cd/eventlistener.yaml

```diff
@@ -169,6 +169,8 @@ spec:
               value:
                 - key: repo
                   expression: body.taskRun.metadata.annotations['tekton.dev/gitURL'].parseURL().path.substring(1)
+                - key: repoUnderscore
+                  expression: body.taskRun.metadata.annotations['tekton.dev/gitURL'].parseURL().path.substring(1).split('/').join('_')
```

Comment:
Uhm, I've tried this with cel-eval and it works well, but I don't know if it's different in the context of an overlay, and the event listener did not seem happy about it

Reply:
```
{"level":"info","ts":"2022-07-04T17:02:01.408Z","logger":"eventlistener","caller":"sink/sink.go:299","msg":"interceptor stopped trigger processing: rpc error: code = InvalidArgument desc = error evaluating cel expression: expression \"body.taskRun.metadata.annotations['tekton.dev/gitURL'].parseURL().path.substring(1).split('/').join('_')\" check failed: ERROR: <input>:1:100: undeclared reference to 'join' (in container '')\n | body.taskRun.metadata.annotations['tekton.dev/gitURL'].parseURL().path.substring(1).split('/').join('_')\n | ...................................................................................................^","eventlistener":"tekton-events","namespace":"default","/triggers-eventid":"08f8e3ef-536a-47f3-af36-e1b29a2f6236","eventlistenerUID":"b37832bb-3e30-4516-9b04-c41d626e9321","/triggers-eventid":"08f8e3ef-536a-47f3-af36-e1b29a2f6236"}
```

Reply:
This is new in triggers v0.20.x - so I need to deploy that in dogfooding first!